enterprise/j2ee.platform/external/binaries-list

```diff
@@ -1 +1,18 @@
-27E9711AA35C39EF455BFD900D544BACB99C0E89 javaee-doc-api.jar
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+CC1024CBFA7C3A9CC84485603A7D458BFD2F3B4A javax:javaee-api:7.0:javadoc
```

Comment:
The artifacts have a different internal structure. The contents of the maven artifact is relocated into `docs/api` in the netbeans artifact. Does it still work?

Reply:
> The artifacts have a different internal structure. The contents of the maven artifact is relocated into `docs/api` in the netbeans artifact. Does it still work?

I searched all the references wherever jar is being used. 
The jar is added as a resource in different modules like below 
<resource>jar:nbinst://org.netbeans.modules.j2ee.platform/docs/javaee-doc-api.jar!/</resource>

It is accessed as
<javadocs>
<lookup path="docs/javaee-doc-api.jar"/>
</javadocs>

So Internal structure of the jar will not cause any issue

Reply:
@matthiasblaesing, Request you to merge the PR

Reply:
@matthiasblaesing, a big thanks for this generous help. I had implemented the same